mne/time_frequency/__init__.py

```diff
@@ -49,14 +55,3 @@
         ],
     },
 )
-from .ar import fit_iir_model_raw
-from .multitaper import dpss_windows, psd_array_multitaper, tfr_array_multitaper
-from .spectrum import (
-    EpochsSpectrum,
-    EpochsSpectrumArray,
-    Spectrum,
-    SpectrumArray,
-    read_spectrum,
-)
-from ._stft import stft, istft, stftfreq
-from ._stockwell import tfr_stockwell, tfr_array_stockwell
```

Comment:
when you merged in main, this wasn't properly integrated into the new lazy loading scheme